charts/hivemq-platform-operator/tests/hivemq_operator_deployment_test.yaml

```diff
@@ -349,8 +358,8 @@ tests:
         runAsUser: 185
     asserts:
       - failedTemplate:
-          errorMessage: |-
-            `runAsNonRoot` is set to `false` but `runAsUser` is not set to `0` (root)
+          errorPattern: |-
```

Comment:
Stupid question: Is there a reason to use `|-` here? I've seen that you refactored a couple of the `errorPattern` into a single line and the majority seems to be like that.

Reply:
Not a stupid question! 
The only reason to use `|-` is because this unittest plugin does not work when the error message to match starts with some special character like ``` in this case. If you see, the `|-` is only applied to those starting with such character. It will fail with:
```
 FAIL   charts/hivemq-platform-operator/tests/hivemq_operator_deployment_test.yaml
        - Execution Error: 
                yaml: line 362: found character that cannot start any token
```
Actually, I'm going to raise this an issue on the Helm unittest plugin.
Another option would be to change the error message in our case.

Reply:
https://github.com/helm-unittest/helm-unittest/issues/499